alerts/templates/email/task_fail.html

```diff
@@ -30,7 +30,7 @@
         <tr><td colspan="2" height="10"></td></tr>
         <tr>
           <td width="90%">
-            <a href="{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
+            <a href="{{$.Settings.Ui.Url}}/{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
```

Comment:
For the server test alerts, yes. I think the way to solve this is to populate the URL field with the whole URL of the logs for test results that come from the test_logs collections. This solution would only work for viewing logs of alerts moving forward, after this change is deployed.